src/main/java/com/SAS/User/Privileges.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * The class represents the privileges of registered user
+ */
+
+package com.SAS.User;
+
+public class Privileges {
+
+    public void edit(String privilege) {
```

Comment:
please add a commnet that says it need to be implemented, or throw "not implemented exception"

Reply:
I add it.

Reply:
we'll change it.